portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,37 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Adds a random fact to the page.
+ */
+function addRandomFact() {
+    const facts = ["I have no sibling", 
+                   "I seldom eat spicy food", 
+                   "I don't like watching thrillers",
+                   "I love drinking milk tea"];
+
+    // Pick a random fact.
+    const factContainer = document.getElementById("fact-container");
+    let currentFact = factContainer.innerText;
+    let fact = getRandomFact(facts, currentFact);
+
+    // Add it to the page.
+    factContainer.innerText = fact;
+} 
+
+/**
+ * Picks a fun fact to be displayed.
```

Comment:
Nit: "Pick ... that is different from the current fun fact" ?

Reply:
Fixed